src/components/sync-connected-sites.tsx

```diff
@@ -177,12 +178,7 @@ const SyncConnectedSitesSection = ( {
 							__( "Studio couldn't connect to this site. <button>Get help ↗️</button>" ),
 							{
 								button: (
-									<Button
-										variant="link"
-										onClick={ () =>
-											getIpcApi().openURL( STUDIO_DOCS_URL_GET_HELP_UNSUPPORTED_SITES )
-										}
-									/>
+									<Button variant="link" onClick={ () => getIpcApi().openURL( docs.sync ) } />
```

Comment:
@wojtekn , I noticed this link was pointing to main Studio docs. I've updated it to point to https://developer.wordpress.com/docs/developer-tools/studio/sync/

<img width="1012" alt="sync" src="https://github.com/user-attachments/assets/db7eef61-da59-4da2-9459-bec595ec6c8b" />

Reply:
Good catch, thanks!